File: objc-1/j00.interface/Rectangle.m
#import "Rectangle.h"

@implementation Rectangle

  /******* INITIALIZERS *******/
  - (Rectangle *) initWithWidth: (double) w height: (double) h {
    if (self = [super init]) { width = w; height = h; }
    return self;
  }

  /******* METHODS *******/
  - (double) calculateArea { return (width * height); };

@end

File: python/h00.override/Rectangle.py
#!/usr/bin/python
from Shape import Shape    # include Shape.py

# class declaration
class Rectangle(Shape):
  ###### INITIALIZER ######
  def __init__(self, width, height):
    super(Rectangle, self).__init__(width, height)

  ###### METHODS ######
  def calculateArea(self):
    return (self.width * self.height)
